src/Appium.Net/Appium/Android/UiAutomator/AndroidUiScrollable.cs

```diff
@@ -0,0 +1,383 @@
+{
+    /// <summary>
+    /// A convenience class to wrap Android UiScrollable method calls.
+    /// </summary>
+    /// <remarks>
+    /// Docs: https://developer.android.com/reference/android/support/test/uiautomator/UiScrollable
+    /// </remarks>
+    public class AndroidUiScrollable : IUiAutomatorStatementBuilder
+    {
+        private readonly StringBuilder _builder;
+
+        /// <summary>
+        /// Creates a new scrollable searcher which will match the first scrollable widget
+        /// in the view.
+        /// </summary>
+        public AndroidUiScrollable()
+            : this(new AndroidUiSelector().IsScrollable(true))
+        {
+        }
+
+        /// <summary>
+        /// Creates a new scrollable searcher which will match the first widget
+        /// which matches the given UISelector.
+        /// </summary>
+        /// <param name="uiSelector">
+        /// A UiSelector that will be used to find the scrollable container
+        /// </param>
+        public AndroidUiScrollable(AndroidUiSelector uiSelector)
+        {
+            _builder = new StringBuilder().AppendFormat("new UiScrollable({0})", 
+                uiSelector.RequireNotNull(nameof(uiSelector)).Build());
+        }
+
+        /// <summary>
+        /// Performs a backwards fling action with the default number of fling steps (5). If the swipe direction
+        /// is set to vertical, then the swipe will be performed from top to bottom. If the swipe direction is set
+        /// to horizontal, then the swipes will be performed from left to right. Make sure to take into account
+        /// devices configured with right-to-left languages like Arabic and Hebrew.
+        /// Maps to the UiScrollable.flingBackward() method.
+        /// </summary>
+        /// <remarks>https://developer.android.com/reference/android/support/test/uiautomator/UiScrollable#flingbackward</remarks>
+        public TerminatedStatementBuilder FlingBackward()
+        {
+            var forkedBuilder = new StringBuilder(_builder.ToString());
+            forkedBuilder.AppendFormat(".flingBackward()");
+            return new TerminatedStatementBuilder(forkedBuilder);
+        }
+
+        /// <summary>
+        /// Performs a forward fling with the default number of fling steps (5). If the swipe direction is set
+        /// to vertical, then the swipes will be performed from bottom to top. If the swipe direction is set to
+        /// horizontal, then the swipes will be performed from right to left. Make sure to take into account
+        /// devices configured with right-to-left languages like Arabic and Hebrew.
+        /// Maps to the UiScrollable.flingForward() method.
+        /// </summary>
+        /// <remarks>https://developer.android.com/reference/android/support/test/uiautomator/UiScrollable#flingforward</remarks>
+        public TerminatedStatementBuilder FlingForward()
+        {
+            var forkedBuilder = new StringBuilder(_builder.ToString());
```

Comment:
`var forkedBuilder = new StringBuilder(_builder.ToString());`

could also be extracted into a separate method, for example:
```
private forkBuilder ()
{
  return new StringBuilder(_builder.ToString());
}


private append (String s)
{
   return new TerminatedStatementBuilder(forkBuilder().AppendFormat(s));
}

public TerminatedStatementBuilder FlingForward()
{
   return append(".flingForward()");
}
....
```